content/applications/marketing/sms_marketing/essentials/sms_essentials.rst

```diff
@@ -176,6 +175,31 @@ is visualized as different graphs and charts, which can be sorted and grouped in
    :align: center
    :alt: Reporting page in SMS Marketing.
 
+.. tip::
+   SMS messages can be sent using server actions in Odoo. To do so, in :ref:`developer mode
+   <developer-mode>`, navigate to :menuselection:`Settings app --> Technical menu --> Automation -->
+   Automated Actions`. Click :guilabel:`Create` or :guilabel:`New` to create a new action.
+
+   Enter an :guilabel:`Action Name` and select a :guilabel:`Model` to implement this action on. Set
+   the :guilabel:`Trigger` to either :guilabel:`On Creation`, :guilabel:`On Update`, :guilabel:`On
+   Creation & Update`, :guilabel:`On Deletion`, :guilabel:`Based on Form Modification`, or
+   :guilabel:`Based on Timed Condition`.
+
+   Based on the selection for the :guilabel:`Trigger`, additional fields may populate. Ensure a
+   model has been selected before creating any record filters.
+
+   Under :guilabel:`Action To Do`, select :guilabel:`Send SMS Text Message`. Next, set the
+   :guilabel:`SMS Template`, and choose whether the SMS message should be logged as a note, by
+   ticking the box next to :guilabel:`Log as note`.
+
+   .. image:: sms_essentials/automated-action-sms.png
+      :align: center
+      :alt: Automated action template with action to do, SMS template and log as note highlighted.
+
+   :guilabel:`Save` the completed automated action to implement the change. Ensure the
+   :guilabel:`Active` toggle is set to *on* for the automated action to run.
```

Comment:
I would flip these sentences, so the user *ends* by saving *after* they've hit the 'Active' toggle
*and* since you *DO* mention the toggle here, you can disregard my previous comment/suggestion about it. 👍